gitbase/templates/statefulset.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
```

Comment:
Please add a README stating that this chart needs at least k8s `1.9.0` to support this.